src/map.h

```diff
@@ -1415,6 +1414,9 @@ class map
         void rad_scorch( const tripoint &p, const time_duration &time_since_last_actualize );
         void decay_cosmetic_fields( const tripoint &p, const time_duration &time_since_last_actualize );
 
+        void process_fire_in_submap( const tripoint &p, const maptile &map_tile, int &part,
```

Comment:
This function does not seem to be implemented.

Reply:
Thanks - that were leftovers of refactor attempt.